src/cli/cli_srp_client.cpp

```diff
@@ -288,14 +543,69 @@ template <> otError SrpClient::Process<Cmd("service")>(Arg aArgs[])
     otError error = OT_ERROR_NONE;
     bool    isRemove;
 
+    /**
+     * @cli srp client service
+     * @code
+     * srp client service
+     * instance:"ins2", name:"_test2._udp,_sub1,_sub2", state:Registered, port:111, priority:1, weight:1
+     * instance:"ins1", name:"_test1._udp", state:Registered, port:777, priority:0, weight:0
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otSrpClientGetServices
+     */
     if (aArgs[0].IsEmpty())
     {
         OutputServiceList(0, otSrpClientGetServices(GetInstancePtr()));
     }
+    /**
+     * @cli srp client service add
+     * @code
+     * srp client service add ins1 _test1._udp 777
+     * Done
+     * @endcode
+     * @code
+     * srp client service add ins2 _test2._udp,_sub1,_sub2 111 1 1
+     * Done
+     * @endcode
+     * @cparam srp client service add @ca{instancename} @ca{servicename} @ca{port} [@ca{priority}] [@ca{weight}]
+     * [@ca{txt}]
```

Comment:
You'll need to line break this different, see https://openthread.io/reference/cli/contribute#long_lists_of_parameters

Doing it this way breaks the rendering of the description beneath